README.md

```diff
@@ -50,59 +43,56 @@ public class SampleApplication {
 
 For a complete example, see `mesos-starter-sample` module.
 
-## Starting the application
-The only required parameter for the scheduler is `mesos.master`. The value of the parameter is passed directly to the Mesos Scheduler Driver which allows the following formats
+# Configuration
+Most of an application's boilerplate is provided my Mesos-Starter. To configure that boilerplate, there are a range of configuration options. This table describes those options, whether they are optional and whether they have a default value.
+
+Command                         | Description                                           | Default               | Required
+---                             | ---                                                   | ---                   | ---
+`spring.application.name`       | Application name                                      |                       | Yes
+`mesos.framework.name`          | Framework name used in Mesos and ZooKeeper            |                       | Yes
+`mesos.master`                  | Path to mesos master                                  |                       | Yes
+`mesos.zookeeper.server`        | IP:PORT of the zookeeper server                       |                       | Yes
```

Comment:
Not required but highly recommended. Only needed if you want to store state in Zookeeper.